### PR TITLE
job board update PR

### PR DESCRIPTION
**Pull request by mlbsmith**
Added the ability to sort jobs by location.

See http://dev.faithtech.com/opportunities/jobs/ for preview.

### CODE REVIEW

File: src/themes/simple/templates/Layout/JobBoardPage.ss [hunk 1]
```diff
@@ -2,17 +2,26 @@
...
+                <% loop $getJobLocations %>
+                    <a data-filter=".$URLSegment">$Title</a>
+                <% end_loop %>
+            </div>
+        </div>
+        <!-- /filter widget -->
+            <div class="row eqh gt40 fs-equalize-element grid">
                 <% loop $Children %>
-                    <div class="col-md-4" style="height: 253px;">
+                    <div class="col-md-4 element-item $loc2class($Location)" style="height: 253px;">
```
Comment (aalkema) [inline src/themes/simple/templates/Layout/JobBoardPage.ss:18]: Do we have classes defined somewhere for each location?  I'm trying to figure out what the loc2class thing is for.

Comment (mlbsmith) [reply src/themes/simple/templates/Layout/JobBoardPage.ss:18]: The `JobPage.php` class has that `loc2class()` method.

Comment (aalkema) [reply src/themes/simple/templates/Layout/JobBoardPage.ss:18]: I saw the method - was more curious about what the class is used for in css (this whole silverstripe world is a weird place)

Comment (mlbsmith) [reply src/themes/simple/templates/Layout/JobBoardPage.ss:18]: Oh true. Yea it shows/hides the div based on which location has been selected in the filter widget above it.

**Comment by aalkema** [comment]
Cool looks good!

**Pull request merged**